dotnet/samples/Concepts/ChatCompletion/Ollama_ChatCompletionStreaming.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.Ollama;
+
+namespace ChatCompletion;
+
+/// <summary>
+/// These examples demonstrate the ways different content types are streamed by Ollama via the chat completion service.
+/// </summary>
+public class Ollama_ChatCompletionStreaming(ITestOutputHelper output) : BaseTest(output)
+{
+    /// <summary>
+    /// This example demonstrates chat completion streaming using Ollama.
+    /// </summary>
+    [Fact]
+    public Task StreamChatAsync()
+    {
+        Assert.NotNull(TestConfiguration.Ollama.ModelId);
+
+        Console.WriteLine("======== Ollama - Chat Completion Streaming ========");
+
+        var chatService = new OllamaChatCompletionService(
+            endpoint: new Uri(TestConfiguration.Ollama.Endpoint),
+            modelId: TestConfiguration.Ollama.ModelId);
+
+        return this.StartStreamingChatAsync(chatService);
```

Comment:
The `StartStreamingChatAsync` is only used by the `StreamChatAsync` sample. I would suggest inline the method unless there are plans to reuse it by other samples.

Reply:
Its being used by the two first Facts on this file.